plutus-benchmark/common/PlutusBenchmark/Common.hs

```diff
@@ -105,12 +112,6 @@ haskellValueToTerm
     :: Tx.Lift DefaultUni a => a -> Term
 haskellValueToTerm = compiledCodeToTerm . Tx.liftCodeDef
 
-
-{- | Convert a de-Bruijn-named UPLC term to a CEK Benchmark -}
-benchTermCek :: Term -> Benchmarkable
-benchTermCek term =
-    nf unsafeRunTermCek $! term -- Or whnf?
-
 {- | Convert a de-Bruijn-named UPLC term to a CEK Benchmark -}
 benchProgramCek :: Program -> Benchmarkable
```

Comment:
Should we fix this too?  It's used in the marlowe and bls12-381-costs benchmarks.

Reply:
Left as future work, I wrote a comment in [PLT-6541](https://input-output.atlassian.net/browse/PLT-6541), there's plenty to do there and I only wanted to unblock my work on costing stuff for the time being.